src/ui/base-page-storyEdit.js

```diff
@@ -274,50 +364,73 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.
+            if (savedStoryData) {
+                // a story was loaded from autosave, update the current story
+                storyEdit.loadStoryContent(savedStoryData);
             } else {
-                fluid.log(fluid.logLevel.WARN, "Story load aborted, no autosaved story was found");
+                // there's no autosaved story, create a new unpublished story
+                storyEdit.createNewStoryOnServer();
             }
         } catch (ex) {
-            fluid.log(fluid.logLevel.WARN, "An error occurred when loading", ex);
+            fluid.log(fluid.logLevel.WARN, "An error occurred while initializing story", ex);
         }
     };
 
     /**
-     * Removes the URL values of all media blocks within a collection of blocks
+     * Updates the storyEditor's story and blockManager with saved story data
      *
-     * @param {Component[]} blocks - a collection of story blocks (sjrk.storyTelling.block)
+     * @param {Object} savedStoryData - story model data loaded from autosave
+     * @param {Component} story - an instance of `sjrk.storyTelling.base.page.storyEdit`
+     * @param {Component} blockManager - the storyEditor's blockManager component
+     * @param {String} sourceName - the name of the Infusion change source for this update
+     */
+    sjrk.storyTelling.base.page.storyEdit.loadStoryContent = function (savedStoryData, story, blockManager, sourceName) {
+        story.applier.change("", savedStoryData, null, sourceName);
+
+        // build the blockUIs from the story content array
+        blockManager.createBlocksFromData(savedStoryData.content);
+    };
+
+    /**
+     * Creates a new story on the server and sets the current story's ID accordingly
      *
-     * @return {Component[]} - a collection of story blocks with nulled-out URL values
+     * @param {String} storySaveUrl - the server URL at which to save a story
+     * @param {Component} story - an instance of `sjrk.storyTelling.story`
+     * @param {String|String[]} storyIdPath - the model path to the story ID
+     * @param {String} sourceName - the name of the Infusion change source for this update
+     * @param {Object} errorEvent - the event to be fired in case of an error
      */
-    sjrk.storyTelling.base.page.storyEdit.clearMediaBlockUrls = function (blocks) {
-        return fluid.transform(blocks, function (block) {
-            if (block.imageUrl) {
-                block.imageUrl = null;
-            } else if (block.mediaUrl) {
-                block.mediaUrl = null;
-            }
+    sjrk.storyTelling.base.page.storyEdit.createNewStoryOnServer = function (storySaveUrl, story, storyIdPath, sourceName, errorEvent) {
+        // The "story created" moment is now
+        story.applier.change("timestampCreated", new Date().toISOString());
+
+        var serverSavePromise = sjrk.storyTelling.base.page.storyEdit.updateStoryOnServer(storySaveUrl, story.model);
```

Comment:
What is in the `story.model` at this point? Is it just the `timestampCreated`? If so, I wonder if it would make more sense to have the server create the time stamp. In terms of the timestamp currently what happens if the `serverSavePromise` is rejected?